README.md

```diff
@@ -10,57 +10,57 @@ eCamp is a web application for planning J+S camps, J+S courses and other camps.
 - Usability on mobile phones - the design is mobile-first.
 - Login via MiData account of the Swiss Guide and Scouts Movement is possible
 - Switching the user interface language, powered by [Lokalise](https://lokalise.com)
-- Formatting texts (bold, italic, etc.) - is prepared, will be activated after the initial release.
+- Formatting texts (bold, italic, etc.)
 
 eCamp v3 is made up of a backend based on the PHP framework API Platform (Symfony), which offers an API, as well as a Vue.js frontend and some other smaller services.
 
-### Want to help?
+### How can I help?
 
 Thanks for helping! There are a few ways to get started.
 
-- Visit our test environment at https://dev.ecamp3.ch. If you discover a bug, [open an issue](https://github.com/ecamp/ecamp3/issues/new) for it.
-- To run the project on your computer for development, follow one of our installation guides:
-  - [Installation on Linux](https://github.com/ecamp/ecamp3/wiki/Development-install-on-linux)
-  - [Installation on Windows + WSL 2](https://github.com/ecamp/ecamp3/wiki/Development-installation-on-Windows)
+- Visit our test environment at [https://dev.ecamp3.ch](https://dev.ecamp3.ch). If you discover a bug, [open a new issue for it](https://github.com/ecamp/ecamp3/issues/new).
+- To run the project locally on your computer, follow one of the installation guides:
+  - [Installation with Docker on Linux](https://github.com/ecamp/ecamp3/wiki/Development-install-on-linux)
+  - [Installation with Docker on Windows + WSL2](https://github.com/ecamp/ecamp3/wiki/Development-installation-on-Windows)
   - [Set up VS Code as your code editor](https://github.com/ecamp/ecamp3/wiki/Development-installation-on-Windows#setting-up-the-ide)
-- If you run into problems during setup or need help, don't worry. Check the [discussions](https://github.com/ecamp/ecamp3/discussions) whether someone already had and solved your problem, and add a [new discussion](https://github.com/ecamp/ecamp3/discussions/new) otherwise.
-- Before you can start coding, read our [contributing guidelines](CONTRIBUTING.md)
-- Study the [documentation in the wiki](https://github.com/ecamp/ecamp3/wiki)
-- Choose an [issue labeled "good first issue"](https://github.com/ecamp/ecamp3/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) and try to solve it
+- If you encounter issues during setup or have questions, don't worry, you're not alone. Check the [discussions](https://github.com/ecamp/ecamp3/discussions) to see if your problem has already been addressed and solved. If not, open a [new discussion](https://github.com/ecamp/ecamp3/discussions/new).
+- Before you start coding, read our [contributing guidelines](CONTRIBUTING.md).
+- Familiarize yourself with the [documentation in the Wiki](https://github.com/ecamp/ecamp3/wiki).
+- Choose an [issue labeled "good first issue"](https://github.com/ecamp/ecamp3/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) and try to solve it.
```

Comment:
I think the setup-wiki-link should be moved to the `CONTRIBUTING.md` and the link to `CONTRIBUTING.md` should pop out a bit more. IMO you should head over to `CONTRIBUTING.md` before taking a look at the wiki. What do you think?

Reply:
Actually, so far your overhaul of the CONTRIBUTING.md in #3874 does not lead to the setup instructions in the wiki. So for now I have left these links in here. If you find a good spot in CONTRIBUTING.md to link to all the setup documentation, I think it would be a good idea to link more prominently from readme to contributing.

But in the current form of CONTRIBUTING.md, I think it isn't bad if someone reads a little in the wiki about the architecture (if we could ever get around to updating those pages) before actually trying to contribute a feature.